config/src/config/state_sync_config.rs

```diff
@@ -113,6 +113,8 @@ pub struct StateSyncDriverConfig {
     pub max_stream_wait_time_ms: u64,
     /// The version lag we'll tolerate before snapshot syncing
     pub num_versions_to_skip_snapshot_sync: u64,
+    /// The time the state sync waits for mempool to repsond to its acks
```

Comment:
nit: Can we change this to: `/// The maximum time (ms) allowed for mempool to ack a commit notification`?

Reply:
Done.